### PR TITLE
Add references.md

### PR DESCRIPTION
**Pull request by btcdrak**
This introduces a central link reference file for _common_ references.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Concept ACK. Would be good to see if this renders correctly.

**Comment by btcdrak** [comment]
@MarcoFalke it renders for me on my local setup.

**Comment by MarcoFalke** [comment]
Ok then, good to merge. I will comment here if something displays incorrectly.

File: _posts/en/pages/2016-01-13-supported-bips.md [hunk 1]
```diff
@@ -8,8 +8,9 @@ permalink: /en/bips/
 share: false
 version: 1
 ---
-Bitcoin Core supports the following [BIPs][BIPS-link].
+Bitcoin Core supports the following [BIP1][].
```
Comment (MarcoFalke) [inline _posts/en/pages/2016-01-13-supported-bips.md]: This displays as "Bitcoin Core supports the following BIP1." on the website.

Comment (btcdrak) [inline _posts/en/pages/2016-01-13-supported-bips.md]: fixed

**Pull request merged**